src/components/main/Scenario.js

```diff
@@ -69,6 +69,11 @@ class Scenario extends React.Component {
     }
 
 	
+    getChildrenMsg = (result) => {
```

Comment:
I could be wrong but isn't the result already in the "staffs[]" so why are we setting the state again here? I am also unclear as to what the function is supposed to be doing what do you mean getChildrenMsg? I'm not sure If I am following your thought process here. Is this supposed to handleUpdateChange?

Reply:
He's using this to add and subtract the quantity for staff members, but I believe this isn't necessary. There is already a function called handleStaffChange, which I believe can be used for adding and subtracting in stafflist.